server/dba/seeders/prod/02-species.js

```diff
@@ -0,0 +1,125 @@
+'use strict';
+
+module.exports = {
```

Comment:
Why are species and their notes two separate tables, is there a scenario in which they will be accessed separately?

Reply:
@lauraGgit species and speciesNotes are oneToMany. There aren't accessed separately though.

Reply:
Also, we might create interface to change the species notes.

Reply:
seems like we could have them as one until the use case suggested that we needed them sepated.